regulations/make.py

```diff
@@ -210,7 +211,12 @@ def buildLanguage(args, language):
 
   pdfName = languageData[language]["pdf"]
 
-  html.html(language, buildDir, pdfName + ".pdf", isTranslation=isTranslation, verbose=args.verbose)
+  srcDir = "translations/" + language if isTranslation else "wca-regulations"
+
+  warnings = check_missing.check_guidelines(language, srcDir)
+  if len(warnings) > 0:
+    print warnings.encode('utf-8')
```

Comment:
I'll try to keep the encode stuff in the check_missing file, here the encode is actually only needed if you want to output to a file (like using `deploy.sh rebuild_regs > log`), for console output it's not necessary (not really sure why).